modules/tinymce/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed a workaround for ensuring stylesheets are loaded in an outdated version of webkit. #TINY-9433
 - Lists in a noneditable root were incorrectly editable using list API commands, toolbar buttons and menu items. #TINY-9458
 - Color picker dialog would not update the preview color if the hex input value was prefixed with `#` symbol. #TINY-9457
+- The Edit Link dialog incorrectly retrieved the URL value when opened immediately after the link insertion. #TINY-7993
```

Comment:
Maybe add ticks or quote around `Edit Link` not sure what the process is there. Maybe look for examples in the changelog.

Reply:
Looked through the CHANGELOG and it varies from italic to using quotes. Feels like the quotes is the way to go. Fixed.